net/gallery-dl/Portfile

```diff
@@ -4,17 +4,17 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        mikf gallery-dl 1.29.5 v
+github.setup        mikf gallery-dl 1.29.2 v
```

Comment:
you're downgrading the version here which is likely not your intention?

Reply:
This is what i get for trying to computer without my glasses on. thanks for catching that.